package.json

```diff
@@ -57,7 +60,7 @@
   },
   "style": "dist/css/bootstrap.css",
   "sass": "scss/bootstrap.scss",
-  "main": "dist/js/bootstrap",
+  "main": "src/js/index",
```

Comment:
That's only possible by importing the modules with `import Tooltip from 'bootstrap/js/some-module-name'`. Importing the modules anywhere (like in `index.js`) evaluates the files as evident in https://codesandbox.io/s/Mjgm6qzAm, rendering `import { SomeModule } from 'bootstrap'` pretty moot as it's always equivalent of `import Bootstrap from 'bootstrap'`.

Userland could adapt some bundler that's smart enough to tree-shake the other modules but even that might be a scretch. Do correct me if I am wrong though.